report/src/presence/bluetooth.tex

```diff
@@ -32,11 +32,14 @@ \subsubsection{Bluetooth Low Energy} \label{sec:bluetooth_low_energy}
 This is because most use cases require LE Coded PHY, a feature of BLE that improves the range and robustness of the wireless communication at the cost of a lower data transfer rate.\cite{BluetoothOverview}\cite{BLE_Regulatory_Aspects_Document}
```

Comment:
```suggestion
This is because most use cases require LE Coded PHY, a feature of BLE that improves the range and robustness of the wireless communication at the cost of a lower data transfer rate.\cite{BluetoothOverview, BLE_Regulatory_Aspects_Document}
```